### PR TITLE
New rebase test

### PR DESCRIPTION
**Pull request by iluxa-c0m**

### CODE REVIEW

**Comment by iluxa-com** [comment]
0000

**Comment by iluxa-com** [comment]
review

**Comment by iluxa-com** [comment]
14-49

**Comment by iluxa-com** [comment]
15-04

File: new-rebase-test.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+The standard Lorem Ipsum passage, used since the 1500s
+
+"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
```
Comment (iluxa-c0m) [inline new-rebase-test.txt]: 3

File: new-rebase-test.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+The standard Lorem Ipsum passage, used since the 1500s
```
Comment (iluxa-c0m) [inline new-rebase-test.txt]: 1

File: new-rebase-test.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+Section 1.10.33 of "de Finibus Bonorum et Malorum", written by Cicero in 45 BC
+
+"At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio. Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus. Temporibus autem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusandae. Itaque earum rerum hic tenetur a sapiente delectus, ut aut reiciendis voluptatibus maiores alias consequatur aut perferendis doloribus asperiores repellat."
```
Comment (iluxa-c0m) [inline new-rebase-test.txt]: 16

File: new-rebase-test.txt (hunk 2 above)
Comment (iluxa-c0m) [inline new-rebase-test.txt]: `


`

File: new-rebase-test.txt [hunk 4]
```diff
@@ -0,0 +1,16 @@
+The standard Lorem Ipsum passage, used since the 1500s
+
+"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+
+--------------
+
++++++++
++++++++
```
Comment (iluxa-com) [inline new-rebase-test.txt]: |`line 12`|
|---|
inline up

Comment (iluxa-com) [inline new-rebase-test.txt:8]: |`line 10`|
|---|
10

File: new-rebase-test.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+The standard Lorem Ipsum passage, used since the 1500s
+
+"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
```
Comment (iluxa-com) [inline new-rebase-test.txt:3]: lorem

File: new-rebase-test.txt [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+Section 1.10.32 of "de Finibus Bonorum et Malorum", written by Cicero in 45 BC
```
Comment (iluxa-com) [inline new-rebase-test.txt]: 1924

File: new-rebase-test.txt [hunk 7]
```diff
@@ -0,0 +1,15 @@
+The standard Lorem Ipsum passage, used since the 1500s
+
+"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+
```
Comment (iluxa-com) [inline new-rebase-test.txt:4]: -Section

**Pull request closed without merging**